libguard/guard_interface.cpp

```diff
@@ -124,6 +124,25 @@ static GuardRecord getHostEndiannessRecord(const GuardRecord& record)
     return convertedRecord;
 }
 
+/**
+ * @brief Function to get the size remaining in the file
+ *
+ * @param[in] size - File size
```

Comment:
Guard file size ?

Reply:
Yes guard file size.

Reply:
I am asking can we you change "File size" into "Guard File Size" - I feel file size is common name and it will give confusion right?

Reply:
Done.